develop/dev-guide-proxysql-integration.md

```diff
@@ -117,7 +117,7 @@ systemctl start docker
 
 </SimpleTab>
 
-### Option 1: Integrate TiDB Cloud with ProxySQL
+### Option 1: Integrate TiDB Cloud (Serverless) with ProxySQL
```

Comment:
L42:
```
- Option 1: [Integrate TiDB Cloud (Serverless) with ProxySQL](#option-1-integrate-tidb-cloud-serverless-with-proxysql)
```

Reply:
Thanks for the catch.  Now it is updated in https://github.com/pingcap/docs/pull/11505/commits/6950d1d67a5d88c196f6ea59499b316bdbe49247.